test/prompt/test_prompt_template.py

```diff
@@ -43,6 +46,35 @@ def test_prompt_template_repr():
     assert str(p) == desired_repr
 
 
+@pytest.mark.unit
```

Comment:
`unit` tests can't make HTTP requests anymore, this will keep failing.

Reply:
K, got it. Will use a custom-node instead..

Reply:
The changes made doesn't imply the need to run a model, we just this need to test `Pipeline._load_or_get_component()` that doesn't even require a `Pipeline` instance to run.

Fetching and loading a model takes time, CI is already way too bloated, we don't need another test that loads models for no reasons.